eq_translations/schema_translation.py

```diff
@@ -13,8 +13,8 @@ def load(self, translation_file_path):
         with open(translation_file_path, encoding="utf8") as translation_file:
             self.catalog = pofile.read_po(translation_file)
 
-            if self.catalog.locale_identifier == "sco_ulster":
-                self.catalog.locale = Locale("eo")
+            if self.catalog.locale_identifier == "ga_IE":
+                self.catalog.locale = Locale("ga")
```

Comment:
Why was this changed?

Reply:
There was a random test_language-eo po, but no JSON to go with it, seemed completely pointless and as I had introduced a proper ga po I switched it over. It will also be easier for test schema translations if needed as Ulster Scots doesn't have one online

Reply:
This was added to support Ulster scots in runner. Babel doesn't support Ulster scots but does support ga_IE hence `eo` is manually set.

Reply:
That does make sense, I weirdly thought this was in a test. I have also added a comment to clarify why we do it